source/chapters/appendix/mixxx_controls.rst

```diff
@@ -3757,6 +3748,16 @@ Controls
    :range: binary, read-only
 
 
+.. mixxx:control:: [EffectRack1_EffectUnitN],loaded_chain_preset
+                   [EqualizerRack1_[ChannelI]],loaded_chain_preset
+                   [QuickEffectRack1_[ChannelI]],loaded_chain_preset
+
+   0-based index of the currently loaded EffectChain preset. `0` is the empty/passthrough
+   preset, `-1` indicates an unsaved preset (default state of `[EffectRack1_EffectUnitN]`).
+
+   :range: integer, 0..:mixxx:coref:`num_chain_presets<[EffectRack1_EffectUnitN],num_chain_presets>` - 1
```

Comment:
the core-ref doesn't seem to work in the render. 
![image](https://github.com/mixxxdj/manual/assets/12380386/8d562fdb-68aa-4e8a-b5ea-82197a7c7a52)

Reply:
oh, that's because of the leading ..
fixed